lage.config.js

```diff
@@ -16,6 +17,8 @@ module.exports = {
     'update-snapshots': ['^update-snapshots'],
     '@fluentui/docs#build': ['@fluentui/react-northstar#build:info'],
     'verify-packaging': ['build'],
+    e2e: [],
```

Comment:
lage is naive, if it doesn't have target definition within config, it executes everything.